docs/htdocs/info/docs/tools/vep/script/vep_options.html

```diff
@@ -892,6 +892,21 @@ <h2 id="output">Output options</h2>
         <div style="padding:2px 0px"><a href="#opt_minimal">--minimal</a></div>
```

Comment:
Mention incompatibility with `--individual_zyg`

```suggestion
        <div style="padding:2px 0px"><a href="#opt_minimal">--minimal</a></div>
        <div style="padding:2px 0px"><a href="#opt_individual_zyg">--individual_zyg</a></div>
```